cfme/configure/access_control.py

```diff
@@ -399,7 +478,7 @@ def is_displayed(self):
         )
 
 
-class DetailsGroupView(ConfigurationView):
+class DetailsGroupView(GroupForm):
```

Comment:
This was an old change that from when I was learning how to manipulate the page object. There was no need for the GroupForm over ConfigurationView as pointed out by @rlbabyuk